src/skin.cpp

```diff
@@ -605,35 +608,39 @@ void SkinInfillAreaComputation::generateGradualInfill(SliceMeshStorage& mesh, un
     }
 }
 
-void SkinInfillAreaComputation::combineInfillLayers(SliceMeshStorage& mesh, unsigned int amount)
+void SkinInfillAreaComputation::combineInfillLayers(SliceMeshStorage& mesh)
 {
-    if (mesh.layers.empty() || mesh.layers.size() - 1 < static_cast<size_t>(mesh.getSettingAsCount("top_layers")) || mesh.getSettingAsCount("infill_line_distance") <= 0) //No infill is even generated.
+    if (mesh.layers.empty() || mesh.layers.size() - 1 < static_cast<size_t>(mesh.settings.get<size_t>("top_layers")) || mesh.settings.get<size_t>("infill_line_distance") == 0) //No infill is even generated.
```

Comment:
should be `get<coord_t>`

this was a bug which I just found...

Reply:
Fixed: db892c43e205d4fc8b3a007c0aba80734693ce55

Merge conflict resolution would've discovered this eventually too, I guess.